hangar-api/pom.xml

```diff
@@ -49,6 +49,12 @@
 			<version>19.0</version>
 		</dependency>
 
+        <dependency>
```

Comment:
The file should be formatted with spaces rather than tabs - I dunno why this one isn't. If you can reformat so it's all spaces, that'd be great.

Who invented \t anyway?!